### PR TITLE
Feat/ras get user

### PR DESCRIPTION
**Pull request by BinamB**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### Improvements
- `/auth/request` now can take `user_id` in body instead of just `token` to check user's access

### CODE REVIEW

File: arborist/server.go [hunk 1]
```diff
@@ -395,8 +401,15 @@ func (server *Server) handleAuthRequest(w http.ResponseWriter, r *http.Request,
...
 	policies := []string{}
+	var username string
+	var clientID string
 	if info != nil {
 		policies = info.policies
+		username = info.username
+		clientID = info.clientID
+	} else {
+		username = authRequestJSON.User.UserId
+		clientID = ""
```
Comment (Avantol13) [inline arborist/server.go:412]: we may eventually want this to also be something we can pass in... can we make a new ticket for the future to do that?

File: docs/openapi.yaml [hunk 2]
```diff
@@ -126,7 +126,15 @@ paths:
...
         the corresponding client will be also checked; only when both the user
-        and the client have permission can the response be positive.
+        and the client have permission can the response be positive. 
+        IMPORTANT NOTE: Under `body` you can now also check authorization of 
+        a user given just the username using the field `user_id` instead of 
+        using `token`. This means that anyone can check anyone's authorization
+        if they have their username. For this reason, this API is not meant to 
+        be fully exposed publically. Arborist does not currently have 
+        authorization for the management of the authorization in the system.
+        The service itself does not provide a way to expose only certain part
```
Comment (Avantol13) [inline docs/openapi.yaml:136]:
```suggestion
        The service itself does not provide a way to expose only certain parts 
```

**Review by Avantol13 — DISMISSED** [review]
left some minor suggestions, overall this looks good

File: arborist/server.go (hunk 1 above)
Comment (BinamB) [reply arborist/server.go:412]: https://ctds-planx.atlassian.net/browse/PXP-9106

**Review by Avantol13 — APPROVED** [review]

**Pull request merged**